.test-infra/jenkins/job_ClusterCredentialsRotation.groovy

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+
+job('Rotate Cluster Credentials') {
+  description('Rotates Certificates and performs an IP rotation for metrics and io-datastores')
+
+  // Set common parameters.
+  commonJobProperties.setTopLevelMainJobProperties(delegate)
+
+  // Sets that this is a cron job.
+  commonJobProperties.setCronJob(delegate, 'H 2 1 */2 *')// At 00:02am every second month.
+
+  steps {
+
+    //Credentials rotation for metrics and io-datastores
+
+    //Set a maintenance window
+    shell('''printf 'yes'| gcloud container clusters update metrics \
```

Comment:
From `gcloud container clusters update --help` I see there is a gcloud global flag so I checked `gcloud --help` and this might simplify the commands here:

```
     --quiet, -q
        Disable all interactive prompts when running gcloud commands. If input
        is required, defaults will be used, or an error will be raised.

        Overrides the default core/disable_prompts property value for this
        command invocation. This is equivalent to setting the environment
        variable CLOUDSDK_CORE_DISABLE_PROMPTS to 1.
```

Would this work?

Reply:
Yes, it works, I avoided to use --quiet cause I thought it would remove the logs, but I tested it and it only auto confirms the command, I removed printf and added --quiet instead, thanks.